tests/python/test_svd.py

```diff
@@ -61,18 +62,16 @@ def run():
                 assert sigma[None][i, j] == test_utils.approx(0)
 
 
-def test_svd():
-    for fp in [ti.f32, ti.f64]:
-        for d in [2, 3]:
+@pytest.mark.parametrize("dim", [2, 3])
+@test_utils.test(default_fp=ti.f32)
```

Comment:
It seems the `fast_math` is missing from the arguments

Reply:
In addition, should we set `require` to `[]`?

Reply:
> It seems the fast_math is missing from the arguments

Yeah, fixed

> In addition, should we set require to []?

No, it is set by https://github.com/taichi-dev/taichi/blob/d799f9ba41094b2c9d5c2c9ecbd01e2acda7e936/tests/test_utils.py#L156-L157

Reply:
> But one existing test [test_disable_fallback](https://github.com/taichi-dev/taichi/blob/master/tests/python/test_test.py#L129) is failing now.

I changed the test to call `ti.init` directly since `test()` doesn't work in a nested scope

Reply:
Neat! Thanks for your great work!